src/js/game/LevelMVC/LevelView.js

```diff
@@ -1159,6 +1159,31 @@ module.exports = class LevelView {
     }
   }
 
+  refreshActionPlane(levelData, indices) {
+    var sprite,
+      blockType;
+
+    for (var index in indices) {
+      if (indices[index] >= 0) {
```

Comment:
If you use `for (let i = 0; i < indices.length; i++)` instead of `for (... in ...)` you shouldn't need this check.  In JS the `for ... in` operator will enumerate all enumerable properties, which includes things like [].length.

Another (good) option would be to use `indices.forEach(index => { ... })`.